themes/default/content/docs/reference/service-api/_index.md

```diff
@@ -0,0 +1,1465 @@
+        "propertyDependencies": {
+          "cidrBlock": null
+        }
+      }
+    ]
+  }
+}
+```
+<!-- ###################################################################### -->
+
+## Stack Tags
+
+### Get Stack Tags
+
+Use [Get Stack](#get-stack) to retrieve a stack's details including its tags.
+
+### Set Stack Tag
+
+```
+POST https://api.pulumi.com/api/stacks/{organization}/{project}/{stack}/tags
+```
+
+#### Parameters
+
+| Parameter | Type | In | Description |
+| --------- | ---------- | ---------- | ---------- |
+| `organization` | string | path | organization name |
+| `team` | string | path | team name |
+| `stack` | string | path | stack name |
+| `name` | string | body | tag name |
+| `value` | string | body | tag value |
+
+#### Example
+
+```bash
+curl \
+  -H 'Accept: application/vnd.pulumi+3' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: token pul-abcdefghijklmnopqrstuvwxyz' \
+  --request POST \
+  --data '{"name":"{tagName}","value":"{tagValue}"}' \
+  https://api.pulumi.com/api/stacks/{organization}/{project}/{stack}/tags
+```
+
+#### Default response
+
+```
+Status: 204 OK
+```
+
+```
+EMPTY RESPONSE BODY
+```
+
+<!-- ###################################################################### -->
+
+### Delete Stack Tag
+
+```
+DELETE https://api.pulumi.com/api/stacks/{organization}/{project}/{stack}/tags/{tagName}
+```
+
+#### Parameters
+
+| Parameter | Type | In | Description |
+| --------- | ---------- | ---------- | ---------- |
+| `organization` | string | path | organization name |
+| `team` | string | path | team name |
+| `stack` | string | path | stack name |
+| `tagName` | string | path | tag name |
+
+#### Example
+
+```bash
+curl \
+  -H 'Accept: application/vnd.pulumi+3' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: token pul-abcdefghijklmnopqrstuvwxyz' \
+  --request DELETE \
+  https://api.pulumi.com/api/stacks/{organization}/{project}/{stack}/tags/{tagName}
+```
+
+#### Default response
+
+```
+Status: 204 OK
+```
+
+```
+EMPTY RESPONSE BODY
+```
+
+## Stack Updates
+
+### List Stack Updates
+
+```
+GET https://api.pulumi.com/api/stacks/{organization}/{project}/{stack}/updates
+```
+
+By default the results are not paginated. You can specify `page` and `pageSize` query parameters to paginate the results.
+> `?pageSize=1&page=1` can be used to return only the most recent update.
+
+#### Parameters
+
+| Parameter | Type | In | Description |
+| --------- | ---------- | ---------- | ---------- |
+| `organization` | string | path | organization name |
+| `project` | string | path | project name |
+| `stack` | string | path | stack name |
+| `page` | number | query | **Optional.** page of the results to return |
+| `pageSize` | number | query | **Optional.** number of results per page |
+| `output-type` | number | query | **Optional.** the response format to return - possible values are `service` or `cli` (default) |
```

Comment:
Ugh, yeah, this is a lame implementation detail bleeding through. Essentially the difference is that the "service" output type includes more data, whereas the CLI does not. But off the top of my head I don't recall the exact details.